solr/bin/solr.cmd

```diff
@@ -429,21 +400,38 @@ IF "%1"=="-a" goto set_jvm_opts
 IF "%1"=="--jvm-opts" goto set_jvm_opts
 IF "%1"=="-j" goto set_addl_jetty_config
 IF "%1"=="--jettyconfig" goto set_addl_jetty_config
+IF "%1"=="--jetty-config" goto set_addl_jetty_config
 IF "%1"=="--noprompt" goto set_noprompt
 IF "%1"=="--no-prompt" goto set_noprompt
+
+REM Skip stop arg parsing if not stop command
+IF NOT "%SCRIPT_CMD%"=="stop" goto parse_general_args
+
+:parse_stop_args
 IF "%1"=="-k" goto set_stop_key
 IF "%1"=="--key" goto set_stop_key
 IF "%1"=="--all" goto set_stop_all
 IF "%1"=="-all" goto set_stop_all
+
+:parse_general_args
+
+REM Print usage of command in case help option included
+IF "%1"=="--help" goto usage
+IF "%1"=="-help" goto usage
+IF "%1"=="-h" goto usage
+IF "%1"=="-usage" goto usage
```

Comment:
we can get rid of any help command that isn't --help or -h

Reply:
I'll wait for https://github.com/apache/solr/pull/2820 to be finished so that I can remove them safely.